+++
hero = "staph.jpg"
pubmed = ""
doi = "10.1101/2025.01.07.631807 "
title = "Metagenomic estimation of absolute bacterial biomass in the mammalian gut through host-derived read normalization"
keywords = ["preprint"]
date = "2025-02-12T09:50:00+01:00"
+++

Absolute bacterial biomass estimation in the human gut is crucial for understanding
microbiome dynamics and host-microbe interactions. Current methods for quantifying
bacterial biomass in stool, such as flow cytometry, qPCR, or spike-ins (i.e., adding
cells or DNA from an organism not normally found in a sample), can be labor-intensive,
costly, and confounded by factors like water content, DNA extraction efficiency, PCR
inhibitors, and other technical challenges that add bias and noise. We propose a simple,
cost-effective approach that circumvents some of these technical challenges: directly
estimating bacterial biomass from metagenomes using bacterial-to-host (B:H) read ratios.
We compare B:H ratios to the standard methods outlined above, demonstrating that B:H
ratios are useful proxies for bacterial biomass in stool and possibly in other
host-associated substrates. We show how B:H ratios can be used to track antibiotic
treatment response and recovery in both mice and humans, which showed 403-fold and
45-fold reductions in bacterial biomass during antibiotic treatment, respectively. Our
results indicate that host and bacterial metagenomic DNA fractions in human stool
fluctuate longitudinally around a stable mean in healthy individuals, and the average
host read fraction varies across healthy individuals by < 8-9 fold. B:H ratios offer a
convenient alternative to other absolute biomass quantification methods, without the
need for additional measurements, experimental design considerations, or machine
learning algorithms, enabling retrospective absolute biomass estimates from existing
stool metagenomic data.

{{< bluesky link="https://bsky.app/profile/gibbological.bsky.social/post/3lfamyp7ihc2s" >}}
