+++
hero = "lego2.jpeg"
pubmed = "40609069"
doi = "10.1371/journal.pcbi.1013253 "
title = "Understanding disease-associated metabolic changes in human colonic epithelial cells using the iColonEpithelium metabolic reconstruction"
keywords = ["new"]
date = "2025-07-03T13:09:49+01:00"
+++

The colonic epithelium plays a key role in the host-microbiome interactions, allowing
uptake of various nutrients and driving important metabolic processes. To unravel
detailed metabolic activities in the human colonic epithelium, our present study focuses
on the generation of the first cell-type-specific genome-scale metabolic model (GEM) of
human colonic epithelial cells, named iColonEpithelium.

GEMs are powerful tools for
exploring reactions and metabolites at the systems level and predicting the flux
distributions at steady state. Our cell-type-specific iColonEpithelium metabolic
reconstruction captures genes specifically expressed in the human colonic epithelial
cells. iColonEpithelium is also capable of performing metabolic tasks specific to the
colonic epithelium. A unique transport reaction compartment has been included to allow
for the simulation of metabolic interactions with the gut microbiome. We used
iColonEpithelium to identify metabolic signatures associated with inflammatory bowel
disease.

We used single-cell RNA sequencing data from Crohn's Diseases (CD) and
ulcerative colitis (UC) samples to build disease-specific iColonEpithelium metabolic
networks in order to predict metabolic signatures of colonocytes in both healthy and
disease states. We identified reactions in nucleotide interconversion, fatty acid
synthesis and tryptophan metabolism were differentially regulated in CD and UC
conditions, relative to healthy control, which were in accordance with experimental
results. The iColonEpithelium metabolic network can be used to identify mechanisms at
the cellular level, and we show an initial proof-of-concept for how our tool can be
leveraged to explore the metabolic interactions between host and gut microbiota.
