+++
title = "Open PhD position: Designing the niche space of the human gut microbiome"
hero = "creative1.jpg"
authors = ["Christian Diener"]
keywords = ["position", "closed", "phd"]
date = "2025-02-11T14:49:59+01:00"
links = { "apply here" = "https://phd-recruiting.medunigraz.at/" }
+++

**Application for this position has concluded. Thank you!**

Virtually every surface of our planet is colonized by microbes and the human body is no
exception. The 38 trillion microbes living in our gut metabolize a large fraction of the
dietary metabolites, pharmaceutical drugs, and xenobiotics [we consume](https://doi.org/10.1038/s41579-020-0433-9) and thus provide
an important interface between the environment and [our blood stream](https://doi.org/10.1038/s42255-022-00670-1).

Whereas many of the species forming the human gut microbiome have been
well-characterized by now, we still know very little about the factors that determine
which microbial species can coexist in any given individual. This currently limits our
understanding regarding engraftment of pathogens and probiotics and how those will
interact with other microbes and host tissues [in the human gut](https://doi.org/10.1038/s41575-024-01023-x).

Metabolic interactions between microbes in complex microbial communities and their host
are a complex ballet of metabolic exchanges and the resulting ecology. The research
project will aim to chip away at this complexity using a combination of wet lab and
computational techniques with the goal to quantify and predict the nutrient uptake
patterns of individual microbial strains and species embedded in complex communities and
varying environments. By studying uptake fluxes and the resulting species phenotypes in
silico, in vitro, ex vivo, and in vivo we aim to provide a resource map for important
microbial species in the human gut to make nutrient uptake and the resulting microbial
phenotypes predictable. Those patterns will then be used to match next generation
probiotics to individual microbiomes by orthogonal niche engraftment or to design
specific synbiotics to enable engraftment in resistant donor microbiomes.

{{< figure
  src="research_statement.webp"
  alt="Outline of the project strategy."
  caption="Strategies used in the project."
>}}

Our research combines a variety of techniques ranging from high throughput wet lab
approaches including [metagenomics, metabolomics](https://doi.org/10.1038/s41564-024-01728-4), and single-cell culturing to
computational methods such as [metabolic modeling](https://doi.org/10.1128/msystems.00606-19), functional association analysis, and
machine learning and thus provides a large degree of flexibility for the applicants
background and role within the project.

## What we offer

We are a new and still fairly small lab at the Medical University of Graz
at the Diagnostic and Research Institute of Hygiene, Microbiology and Environmental
Medicine. We value diversity, creativity, communication and research integrity. The team
spans a wide array of backgrounds and abilities, and lab members are encouraged to share
their knowledge and experiences. Our goal is to support all lab members in accomplishing
their individual career goals, while also conducting awesome science with a clear path
into clinical application. We collaborate with research groups all around the globe and
invite you to be a part of this as well. More info about the lab can be found [here](https://dienerlab.com).

## What about you?

Do you have some experience in working with the microbiome in the human
gut or other environments? Are you curious and excited about projects that combine
microbial ecology, systems biology, and clinical samples or data? Do you have excellent
English communication skills and love to discuss your research with your peers?

Then you will fit in perfectly with our lab.

## Skills

Additionally, the following general skills synergize great with the research project, and you will be trained in these
during the project. Any prior knowledge is, of course, an advantage.

*Dry Lab*

- Analysis of sequencing data.
- Analysis of mass spectroscopy or NMR data.
- Metabolic modeling (in Python).
- Statistics (in R).

*Wet Lab*

- Isolation and anaerobic culturing of microbial strains from various biofluids.
- Culturing of microbial consortia.
- Sample preparation and library prep for high throughput sequencing and/or metabolomics.

